src/utils.ts

```diff
@@ -1860,15 +1860,47 @@ export class newText extends newShape {
   }
 }
 
+export interface pointStyle {
+  size?: number, fillStyle?: string, strokeStyle?: string, marker?: string, markerOrientation?: string
```

Comment:
I am surprised the linter let you put this on one line.

Usually, there is one attribute per line

Reply:
I think the integrated linter i choose is not the best

I changed for what you ask